tests/e2e/docs/vanilla_cluster_setup.md

```diff
@@ -65,6 +65,7 @@ list of datastore URLs where you want to deploy file share volumes. Retrieve thi
 
     # For VCP to CSI migration tests following are needed as well
     export SHARED_VSPHERE_DATASTORE_NAME="vsanDatastore"
+    export ESX_TEST_HOST_IP="<esx_host_ip>"  # for static provisioning tests
```

Comment:
Isn't there a better way to choose a esx host through govmomi ?
For ex:

```
export 'GOVC_URL=https://Administrator@vsphere.local:Admin!23@<VC-IP>'
export GOVC_INSECURE=1
$ govc ls /datacenter/host/cluster1
/datacenter/host/cluster1/Resources
/datacenter/host/cluster1/<esx_host_ip-1>
/datacenter/host/cluster1/<esx_host_ip-2>
/datacenter/host/cluster1/<esx_host_ip-3>
```

can we use existing methods to select a esx host ?

Reply:
All the esx host in the given VC may not be part of the cluster we are trying to use. 
Also, in case of a stretched vsan cluster, we should not be using witness host.
So, thought of keeping it simple.

Reply:
Got it